_config.yml

```diff
@@ -1,7 +1,8 @@
 name: ROS2 Design
 description: "Distilled design documents related to the ROS 2.0 effort"
 
-baseurl: http://design.ros2.org
+baseurl: ""
+url: http://design.ros2.org
```

Comment:
That must have changed (again), because that wasn't the case before. Just make sure double check that the live site is working correctly after merging.